### ISSUE TITLE
checkra1n says Booting, and doesn't stop, have to use pkill checkra1n to close

### ISSUE THREAD

**Issue by marleyqp** [checkra1n/BugTracker#1158]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 7 Plus (GSM)
2. On what version of iOS is it?
iOS 13.3
3. What version of checkra1n are you using?
beta version 0.10.1
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Ubuntu 18.04 LTS
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-a to Lightning

**What are the steps to reproduce the issue?**
1. Run the checkra1n jailbreak
2. Wait until the end when it says Booting
...

**What do you expect, and what is happening instead?**
Expect so say All Done, except it's stuck on Booting

**Does the issue also occur if you enable Safe Mode?**
N/A

**Any other info, error logs, screenshots, ...?**

**Comment by nergzd723** [comment]
Did the jailbreak work or not?

**Comment by marleyqp** [comment]
Yes the jailbreak goes through with success. Sometimes it will say ‘All Done’ or get stuck on ‘Booting’ or through me a USBMUXD error

**Comment by nergzd723** [comment]
> Yes the jailbreak goes through with success. Sometimes it will say ‘All Done’ or get stuck on ‘Booting’ or through me a USBMUXD error

That *probably* means that checkra1n doesn't recognize the device after it was jailbroken.
